### PR TITLE
Ch cvp vxlan lab2

### PR DESCRIPTION
**Pull request by coreyhines**
I want to see if we can add Joe Recchia aas a reviewer for this PR as I am incorporating a lot of feedback from his review of the first merge.

### CODE REVIEW

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 1]
```diff
@@ -18,21 +18,46 @@ VxLAN
...
+* Click on **VXLAN** in the left selection column under **Switching**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-pre.png
     :align: center
     :width: 50%
 
-* **Note:** **leaf3** currently has no VXLAN configuration
+* **Note:** leaf3 currently has no VXLAN configuration
+
+* Click on **Topology View**
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:31]:
```suggestion
* Click on **Topology** in the navigation bar at the top of the page 
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 2]
```diff
@@ -18,21 +18,46 @@ VxLAN
...
+* **Note:** leaf3 currently has no VXLAN configuration
+
+* Click on **Topology View** 
+* Click the **Link Overlay** dropdown in the left selection column
+
+.. image:: images/cvp-vxlan/leaf3-vxlan-before.png
+    :align: center
+    :width: 50%
+
+* Click the **VXLANs** selection, also click and view **VLANs** selection
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:38]:
```suggestion
* Click the **VXLANs** selection, also click and view the **VLANs** selection
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 3]
```diff
@@ -18,21 +18,46 @@ VxLAN
...
+.. image:: images/cvp-vxlan/leaf3-vxlan-vni-before.png
+    :align: center
+    :width: 50%
 
 3. Create the VXLAN configlet
 
-* click on Provisioining, click on configlets in the left selection column
-* click the **+** sign in the Configlets list toolbar
-* create a configlet called Leaf3-VXLAN-Lab-Full-user
+* Click on **Provisioning**, click on configlets in the left selection column
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:53]:
```suggestion
* Click on **Provisioning**, click on **Configlets** in the left selection column
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 4]
```diff
@@ -59,34 +84,34 @@ VxLAN
...
     :width: 50%
 
-* validate configlet syntax on **leaf3**
+* Validate configlet syntax on **leaf3**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet-validate.png
     :align: center
     :width: 50% 
 
 4. Assign VXLAN configlet to **leaf3**
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:99]:
```suggestion
4. Assign the VXLAN configlet to **leaf3**
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 5]
```diff
@@ -59,34 +84,34 @@ VxLAN
...
 
-* Click on Provisioning, click on **Network Provisioning** in left selection column
-* right click on Leaf3, click on manage configlets, search for Leaf3-VXLAN 
+* **Click on Provisioning**, then click on **Network Provisioning** in left selection column
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:101]:
```suggestion
* Click on **Provisioning**, then click on **Network Provisioning** in the left selection column
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 6]
```diff
@@ -59,34 +84,34 @@ VxLAN
...
+* right click on **leaf3**, Click on manage->Configlets, search for ``Leaf3-VXLAN``
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:102]:
```suggestion
* Right click on **leaf3**, Click on **Manage->Configlets** and then search for ``Leaf3-VXLAN``
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 7]
```diff
@@ -98,63 +123,63 @@ VxLAN
     :align: center
     :width: 50% 
 
-* Click **save** on the Network Provisioning main view
-    .. note: a **Task** will be generated
+* Click **Save** on the **Network Provisioning** main view
+   **Note:** a **Task** will be generated
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:127]:
```suggestion
   **Note:** a Task will be generated
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 8]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet-main-save.png
     :align: center
     :width: 50% 
 
 5. Create a **Change Control** with the generated Task
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:133]:
```suggestion
5. Create a Change Control with the generated Task
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 9]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
 
-* click **Tasks** from the left selection column
+* Click **Tasks** from the left selection column
 
-* click the checkbox next to the generated task
+* Click the checkbox next to the generated task from the pool of **Assignable Tasks**
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:137]:
```suggestion
* Click the checkbox next to the generated task from the pool of Assignable Tasks
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 10]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-task.png
     :align: center
     :width: 50% 
 
-* click **+** Create Change Control with 1 Task
+* Click **+** Create Change Control with 1 Task
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:143]:
```suggestion
* Click **+ Create Change Control with 1 Task**
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 11]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
 
-* click **Review and Approve** on the Change Control that was created
+* Click **Review and Approve** on the newly created **Change Control**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-review-approve.png
     :align: center
     :width: 50% 
 
-* click **Execute Change Control** in upper right of the UI
+* Click **Execute Change Control** in upper right of the UI
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:155]:
```suggestion
* Click **Execute Change Control** towards the upper right of the UI
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 12]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-execute-confirm.png
     :align: center
     :width: 50% 
 
-6. Verify VLXAN operation with CVP Telemetry
+6. Verify VXLAN operation with CVP Telemetry
 
-* from **Device Inventory** click on **leaf3**
-* click on VXLAN in the left selection column under Switching
+* From **Device** page **Inventory** click on **leaf3**
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:169]:
```suggestion
* From the **Device** page inventory table, click on **leaf3**
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 13]
```diff
@@ -98,63 +123,63 @@ VxLAN
...
+* Click on VXLAN in the left selection column under **Switching**
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:170]:
```suggestion
* Click on **VXLAN** in the left selection column under **Switching**
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst [hunk 14]
```diff
@@ -167,20 +192,40 @@ VxLAN
...
+* **Note:** You will see the local MAC Address of Host2 on Port-Channel 4 and the remote MAC Address of Host1 showing port ``Vxlan1``
+
+* Click on **Topology View** 
+* Click the **Link Overlay** dropdown in the left selection column
+
+.. image:: images/cvp-vxlan/leaf3-vxlan-before.png
+    :align: center
+    :width: 50%
+
+* Click the **VXLANs** selection, also click and view **VLANs** selection
```
Comment (jrecchia1029) [inline topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:218]:
```suggestion
* Click the **VXLANs** selection, also click and view the **VLANs** selection
```

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst (hunk 11 above)
Comment (coreyhines) [reply topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:155]: I am going to reject this suggestion, I think it is even more vague by adding "towards"

**Comment by coreyhines** [comment]
Rob, this commit/merge has been reviewed by Joe Recchia, we are ready for merge.

File: topologies/datacenter-latest/labguides/source/cvp-vxlan.rst (hunk 9 above)
Comment (coreyhines) [reply topologies/datacenter-latest/labguides/source/cvp-vxlan.rst:137]: Where does the notion, come from that bolding is only for clickable items? I am going to attempt to create a ATD labguide style guide to formalize the conventions we are using.

**Pull request merged**